src/Logger/Adapter/AppSignal.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Utopia\Logger\Adapter;
+
+use Exception;
+use Utopia\Logger\Adapter;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+
+// Reference Material
+// https://docs.appsignal.com/api/public-endpoint/errors.html
+
+class AppSignal extends Adapter
+{
+    /**
+     * @var string (required, can be found in Appsignal -> Project -> App Settings -> Push & deploy -> Push Key)
+     */
+    protected string $apiKey;
+
+    /**
+     * Return unique adapter name
+     *
+     * @return string
+     */
+    public static function getAdapterName(): string
```

Comment:
```suggestion
    public static function getName(): string
```

The scope already suggests this is the adapter name.

Reply:
Renamed all to `getName`.